docs/source/api/asns.rst

```diff
@@ -32,11 +32,20 @@ Request Structure
 -----------------
 .. table:: Request Query Parameters
 
-	+----------------+----------+-----------------------------------------------------------------------------------------------------+
-	| Parameter      | Required |                                 Description                                                         |
-	+================+==========+=====================================================================================================+
-	|   cachegroup   | no       | An integral, unique identifier for a Cache Group - only ANSs for this Cache Group will be returned. |
-	+----------------+----------+-----------------------------------------------------------------------------------------------------+
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| Parameter  | Required | Description                                                                                         |
+	+============+==========+=====================================================================================================+
+	| cachegroup | no       | An integral, unique identifier for a Cache Group - only ANSs for this Cache Group will be returned. |
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| orderby    | no       | Choose the ordering of the results - must be the name of one of the fields of the objects in the    |
+	|            |          | ``response`` array                                                                                  |
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| sortOrder  | no       | Changes the order of sorting. Either ascending (default) or descending ("desc")                     |
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| limit      | no       | Choose the number of results to return                                                              |
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| offset     | no       | The number of results to skip before beginning to return results                                    |
```

Comment:
What happens if the offset >= the number of possible results?

Reply:
Internal Server Error

Reply:
I forgot to add - you need a `limit` parameter in order to use `offset`

Reply:
Can you add that restriction in like a footnote or something?